jobs/delorean/3scale/rc/discovery.yaml

```diff
@@ -35,6 +35,11 @@
           default: 'pkg/3scale/amp/auto-generated-templates/productized_templates/amp'
           description: '[REQUIRED] directory where openshift templates are located'
           read-only: true
+      - string:
+          name: 'registryProjectIDs'
+          default: '3scale-amp'
```

Comment:
This used to be 3scale-amp25, but we want this to be less specific to a version. The search will end up returning more images, but since we check them anyway it does no harm and will work for all releases, assuming it always follows this pattern (i.e.  3scale-amp26,  3scale-amp27 etc..)